### ISSUE TITLE
question about loss

### ISSUE THREAD

**Issue by weiyang112** [jrzaurin/pytorch-widedeep#13]
hello,i want to know why the input need to be "input = torch.cat([1 - input, input], axis=1)",what the meaning of 1-input. the input should be original input that not the sigmoid result,because you use F.binary_cross_entropy_loss_withlogits instead of binary_cross_entropy.

**Comment by jrzaurin** [comment]
Hey @weiyang112 

you are right, that is a bug. Is ok for the multiclass problem, but for a binary is wrong. 

I have fixed it in master. 

There is a new version coming and that will change. It will be clearer as no `sigmoid` will be applied before using the Focal Loss. 

Many thanks!

**Comment by jrzaurin** [comment]
The loss is still not correct. 

Bug fix on its way

**Comment by jrzaurin** [comment]
Hi @weiyang112 

The Implementation of the Focal Loss is now been fixed in the new release (0.4.1) 

Going to close this

Thanks again for pointing that out
